universal-application-tool-0.0.1/app/controllers/admin/TrustedIntermediaryManagementController.java

```diff
@@ -20,7 +22,18 @@ public TrustedIntermediaryManagementController(
   }
 
   @Secure(authorizers = Authorizers.Labels.UAT_ADMIN)
-  public Result index() {
-    return ok(listView.render(userRepository.listTrustedIntermediaryGroups()));
+  public Result index(Http.Request request) {
+    return ok(listView.render(userRepository.listTrustedIntermediaryGroups(), request));
+  }
+
+  @Secure(authorizers = Authorizers.Labels.UAT_ADMIN)
+  public Result newGroup(Http.Request request) {
```

Comment:
(blocking)  this method be called `create` to match all the other controllers

Reply:
Done!